Source/Private/ASDisplayNodeInternal.h

```diff
@@ -34,7 +34,6 @@ NS_ASSUME_NONNULL_BEGIN
 @protocol _ASDisplayLayerDelegate;
 @class _ASDisplayLayer;
 @class _ASPendingState;
-@class ASSentinel;
```

Comment:
This was just a leftover – the ASSentinel class was removed in favor of atomic integers.